doc/man7/provider-serializer.pod

```diff
@@ -0,0 +1,180 @@
+=pod
+
+=head1 NAME
+
+provider-serializer - The SERIALIZER library E<lt>-E<gt> provider functions
+
+=head1 SYNOPSIS
+
+ #include <openssl/core_numbers.h>
+
+ /*
+  * None of these are actual functions, but are displayed like this for
+  * the function signatures for functions that are offered as function
+  * pointers in OSSL_DISPATCH arrays.
+  */
+
+ /* Functions to serialize object data */
+ int OP_serializer_serialize_data(void *provctx, const OSSL_PARAM *data,
+                                  BIO *out);
+ int OP_serializer_serialize_object(void *provobj, BIO *out);
```

Comment:
I notice this is write serialization only. Are you planning read serialization in some later PR?